app/views/groups/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<header>
+  <h1>Categories</h1>
+</header>
+<main class="g-bg">
+  <section>
+    <% if @groups.empty? %>
+      <div class="empty">
+        <p>Please add a new category</p>
+      </div>
+    <% end %>
+    <ul class="category-list">
+      <% @groups.each do |group| %>
+        <% total_amount = total_amount(group) %>
+        <li class="w-bg">
+          <%= link_to group_purchases_url(group), data: { turbo: false }, class: 'item' do %>
+            <div class="category-card">
+              <div class="category-image-info">
+                <%= image_tag(check_url(group.icon), class: 'icon', alt: 'group icon') %>
+                <div class="category_info">
+                  <span class="name"><%= group.name %></span>
+                  <span class="date"><%= group.created_at.strftime('%d %b %Y') %></span>
+                </div>
+              </div>
+              <div class="category-amount">
+                <span class="amount">$<%= total_amount %></span>
+              </div>
+            </div>
+          <% end %>
+        </li>
+      <% end %>
+    </ul>
+  </section>
+  <section class="below-form">
+    <%= link_to "Add a Category", new_group_path, class: "btn", data: { turbo: false }  %>
+  </section>
+</main>
```

Comment:
- [ ] As per project requirements, I highly encourage you to stick to the given design as much as possible. Please modify the colors and positions of the elements. Ex: the add category button is a green button at the bottom of the mobile view taking the full width of it. 😉 

![RorBudgetApp - Google Chrome 12_9_2023 1_19_59 PM](https://github.com/firepicaso/ror-budget-app/assets/47625407/76b04d0f-018a-483a-b018-26d7a7fffa20)